### sensor model parameters ###

tau: 0.2 # [m]
r_prob: 0.8 # [m]
below_r_prob: 0.2 # [m]
